src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -2,25 +2,82 @@
 
 import org.springframework.stereotype.Service;
 
+/**
```

Comment:
4. **Javadoc Comments**:
   - Adding Javadoc comments above each method can help provide better documentation for the functionality, parameters, and potential exceptions thrown.